react/features/deep-linking/components/DeepLinkingMobilePage.js

```diff
@@ -110,14 +136,17 @@ class DeepLinkingMobilePage extends Component<*, *> {
                                 { app: NATIVE_APP_NAME })
                         }
                     </p>
-                    <a href = { _URLS[Platform.OS] }>
+                    <a
+                        href = { _URLS[Platform.OS] }
+                        onClick = { this._onDownloadApp } >
```

Comment:
Now that I look at this, I wonder why the button doesn't have an onClick instead...